platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleRoot.java

```diff
@@ -38,13 +38,13 @@ public interface MerkleRoot extends MerkleInternal {
      * @return the platform state
      */
     @NonNull
-    PlatformState getPlatformState();
+    PlatformStateAccessor getPlatformStateAccessor();
```

Comment:
Keeping method name as `getPlatformState()`, just changing the return type, would reduce the number of changes in this PR.

Reply:
Agreed